source/administration/backup.rst

```diff
@@ -59,3 +59,52 @@ A properly deployed high availability setup automatically switches over to a red
 
 A "complete" disaster recovery solution would protect against both real-time hardware failures using high availability, data corruption failures using automated, and failures of the primary data center by offering both offsite backup and offsite redundant infrastructure. Because the complexity of a full disaster recovery solution is high, it is common for customers to consider trade-offs in cost and complexity relative to the anticipated risks and target recovery times.
 
+Failover from single-sign-in outage 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When using single-sign-on with Mattermost Enterprise Edition an outage to your SSO provider can cause a partial outage on your Mattermost instance. 
+
+**What happens during an SSO outage?**
+
+_Most people can still log in_ - By default, when a user logs in to Mattermost they receive a session token lasting 30 days (the duration can be configured in the System Console). During an SSO outage, users with valid session tokens can continue to using Mattermost uninterrupted. 
+
+_Some people can't log in_ - During an SSO outage, there are two situations under which a user cannot log in: 
+
+a) Users who's session token expires during the outage
+
+b) Users trying to log in to new devices 
+
+In each case, the user cannot reach the SSO provider, and cannot log in. In this case, there are several potential mitigations: 
+
+1) Configure your SSO provider for high availability 
+
+If you're using a self-hosted single-sign-on provider like Active Directory, several options are available for `high availability configurations that protect your system from unplanned outages. <https://docs.microsoft.com/en-us/microsoft-identity-manager/pam/high-availability-disaster-recovery-considerations-bastion-environment>`_
```

Comment:
I don't think AD or LDAP is a kind of SSO, maybe refer here to OAuth 2.0 or SAML as SSO's